media/css/pocket/includes/_platform-nav.scss

```diff
@@ -2,109 +2,115 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+$image-path: '/media/protocol/img';
+
+@use '../utils/variables' as pocket;
+
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/base/elements'; // this probably needs to be in base pocket style imports
```

Comment:
Issue (possibly blocking): is this making headings render using Zilla?

![image](https://user-images.githubusercontent.com/400117/201929939-af336cd9-7193-42e0-bbb9-2ece97a38862.png)

Reply:
Also notice some changes to heading margins which affect smaller screen sizes
![image](https://user-images.githubusercontent.com/400117/201930221-fda24388-e24d-4e5c-9967-d1f4f736274c.png)